config/settings.yml

```diff
@@ -89,6 +89,7 @@ feedback:
   link: https://forms.office.com/pages/responsepage.aspx?id=yXfS-grGoU2187O4s0qC-SKECobyE75EtuJMp8rYxZtURTNaTTJaTVhBQlQzM1RESTJDVlBERk1JQS4u
 
 features:
+  api_summary_content_change: false
```

Comment:
If you ask why a feature flag for a content change on the API? 

Thousands of courses will be updated in Apply (confirmed already in the DB) and maybe on Register too.

So I optin for the feature flag approach (see PR description for more info on the plan).

for you awareness @Nitemaeric